explore_prompts/requirements.txt

```diff
@@ -1,3 +1,4 @@
+# TODO remove all this, probably along with all of repo/expore_prompts
```

Comment:
@callummcdougall I don't think I need this explore_prompts directory at all. Can we delete?

Reply:
Yep, if you don't need anything from it, it can be fully deleted